dbos/dbos.py

```diff
@@ -550,15 +554,17 @@ def temp_wf(*args: Any, **kwargs: Any) -> Any:
 
         return decorator
 
-    def communicator(self) -> Callable[[Communicator], Communicator]:
+    def communicator(
```

Comment:
Let's split up the config and not pass it around--no need to

Reply:
Done